classes/game.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../item'
+class Game < Item
```

Comment:
Optional.
As per the UML diagram, a Game belongs to the Item class, and every item should possess an author, label, and genre. However, in your case Game for the author, and that too it accepts blank input. Since previous reviewers didn't raise this as required, I am marking this as optional.